flow/layers/default_layer_builder.cc

```diff
@@ -37,12 +37,13 @@ DefaultLayerBuilder::~DefaultLayerBuilder() = default;
 void DefaultLayerBuilder::PushTransform(const SkMatrix& sk_matrix) {
   SkMatrix inverse_sk_matrix;
   SkRect cullRect;
-  if (sk_matrix.invert(&inverse_sk_matrix)) {
+  // Perspective projections don't produce rectangles that are useful for
+  // culling for some reason.
```

Comment:
do we know the reason?

Reply:
no

Reply:
Can you file a bug with the skia folks? Just to let them know, not high priority. Maybe there's a good reason (in which case we can update the comment), maybe it's a real bug.

Reply:
(and add a link to the bug in this comment)

Reply:
sounds good, I'll update with a issue when it's filled.

Reply:
If you file a Skia bug, a reduced test case in https://fiddle.skia.org/ gets most actionable work items.

Reply:
Do they have a public issue tracker somewhere?

Reply:
http://go/newskbug

Reply:
Oh nvm I see the link